packages/react-openapi/src/OpenAPISchema.tsx

```diff
@@ -92,6 +93,9 @@ export function OpenAPISchemaProperty(
                             className="openapi-schema-description"
                         />
                     ) : null}
+                    {schema.example ? (
+                        <span className="openapi-schema-example">Example: {schema.example} </span>
```

Comment:
We should use a `<code>` or `<pre>` around the value.

Also, how does it work if my example is not a primitive?

ex:

```
example:
  propA: a
  propB: b
```

My take is that it'll render `[object Object]` which is bad.

Reply:
Are you doing the changes here?

Reply:
done. this is what it looks like now
<img width="649" alt="Screen Shot 2024-05-22 at 11 49 45 AM" src="https://github.com/GitbookIO/gitbook/assets/140522517/6966c555-9854-4c83-b45c-0fec50fa764c">